app/controllers/group/register_controller.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2021, Efficiency-Club Bern. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+class Group::RegisterController < CrudController
```

Comment:
```suggestion
class Groups::SelfRegistrationController < CrudController
```
use same term everywhere for this feature. 

maybe also name route like this? groups/self_registration

to me the term self_registration makes it more obvious